packages/esm-commons-lib/src/dashboards/createDashboard.tsx

```diff
@@ -3,18 +3,25 @@ import React from 'react';
 import PatientExtensionRenderer from '../components/extension-conditional-renderer/patient-based-extension-renderer';
 import { DashboardLinkConfig } from '../types';
 import { DashboardExtension } from './DashboardExtension';
+import { BrowserRouter } from 'react-router-dom';
 
 export const createDashboardLinkWithCustomTitle = (db: DashboardLinkConfig) => {
   return ({ basePath }: { basePath: string }) => {
-    return <DashboardExtension title={db.title} basePath={basePath} linkText={db.linkText} />;
+    return (
```

Comment:
we might need to consider moving some of these changes to the community patient chart to keep up with the changes being done